browser/js/common/directives/snippet/snippet.html

```diff
@@ -45,7 +45,7 @@
                 <div class="snippetImages img-wrap" ng-repeat="(index,collaborator) in collaborators track by $index">
                     <span ng-show="type!=='report'" ng-click="removeCollaborator(collaborator.$id)" class="close expansion"><i class="material-icons"> clear</i></span>
                     <md-tooltip>{{collaborator.first_name}} {{collaborator.last_name}}</md-tooltip>
-                    <img class="collaborator-avatar" ng-src="{{ collaborator.photoUrl || '/files/default-profile.png' }}">
+                    <img class="collaborator-avatar" ng-src="{{ collaborator.photoUrl || '/files/default-profile.png' }}" ng-cloak>
```

Comment:
Yeah if you find yourself repeating this a lot it could make a lot of sense to just always store a default value with the instance.